mmv1/third_party/terraform/services/dialogflowcx/resource_dialogflowcx_flow_test.go

```diff
@@ -341,11 +343,26 @@ func testAccDialogflowCXFlow_full(context map[string]interface{}) string {
       audio_export_gcs_destination {
         uri = "${google_storage_bucket.bucket.url}/prefix-"
       }
+      speech_settings {
+        endpointer_sensitivity        = 30
+        no_speech_timeout             = "3.500s"
+        use_timeout_based_endpointing = true
+        models = {
+          name : "wrench"
+          mass : "1.3kg"
+          count : "3"
+        }
+      }
       dtmf_settings {
         enabled      = true
         max_digits   = 1
         finish_digit = "#"
       }
+      logging_settings {
+        enable_stackdriver_logging     = true
```

Comment:
Can you set some of these to `false` in a separate step? Sometimes we see issues setting booleans to false values

Reply:
Or is that impossible to verify based on the ignore_read functionality? Have you tested setting the false value and seeing that it correctly sets it in the API?

Reply:
Hi @slevenick, I tested setting all values to true/false and it all works. The only issue is that one of the  fields `enableInteractionLogging ` automatically set to true if you completely remove `logging_settings {}` block which triggers permadiff. I have internal bug open and will keep an eye on it.
Indeed given that we have `ignore_read`, testing change to one of the fields does not make sense, but I tested it by hands back and force with curl and gcloud to make sure fields change after change to the TF config